vllm/engine/arg_utils.py

```diff
@@ -767,6 +769,18 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
             default=None,
             help="The configurations for speculative decoding. Should be a "
             "JSON string.")
+        speculative_group.add_argument(
+            "--draft-tensor-parallel-size",
+            type=int,
+            default=None,
+            help="Number of tensor parallel replicas for the draft model. "
+            "Only used with speculative decoding.")
```

Comment:
any reason we wouldn't always want to auto-detect?

Reply:
removed the arg to disable auto-detection